src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -641,6 +641,51 @@
     <value>long run operation</value>
   </data>
   <data name="PX1072Title_CreateSharedGraphVariable" xml:space="preserve">
-    <value>Creating a graph for a BQL query is a resource consuming operation. To decrease the number of these operations, create a variable with a shared graph that can be reused by BQL queries.</value>
+    <value>Creating a graph for a BQL query is a resource consuming operation. To decrease the number of these operations, create a variable with a shared graph that can be reused by BQL queries</value>
+  </data>
+  <data name="PX1062MessageFormat" xml:space="preserve">
+    <value>Graphs and graph extensions cannot declare static {0}</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Fields" xml:space="preserve">
+    <value>mutable fields</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Properties" xml:space="preserve">
+    <value>mutable properties</value>
+  </data>
+  <data name="PX1062FixMakeReadOnlyFormat" xml:space="preserve">
+    <value>Make the {0} read-only</value>
+  </data>
+  <data name="PX1062FixFormatArg_Field" xml:space="preserve">
+    <value>field</value>
+  </data>
+  <data name="PX1062FixFormatArg_Property" xml:space="preserve">
+    <value>property</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Views" xml:space="preserve">
+    <value>views</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Actions" xml:space="preserve">
+    <value>actions</value>
+  </data>
+  <data name="PX1062FixMakeNonStaticFormat" xml:space="preserve">
+    <value>Remove static from the {0}</value>
+  </data>
+  <data name="PX1062FixFormatArg_View" xml:space="preserve">
+    <value>view</value>
+  </data>
+  <data name="PX1062FixFormatArg_Action" xml:space="preserve">
+    <value>action</value>
+  </data>
+  <data name="PX1063Title" xml:space="preserve">
```

Comment:
@EAndrosova  please review diagnostic and code fix messages for PX1063 and PX1064

Reply:
Please change to the following (too many nouns in a row + Acumatica is the company name so we should specify ERP)
The declaration of the exception class does not contain a serialization constructor. This will cause an incorrect deserialization of the exception data and will lead to runtime errors in Acumatica ERP.
The declaration of the exception class introduces new serializable fields but does not declare an override for the GetObjectData method. This will cause an incorrect deserialization of the exception data and will lead to runtime errors in Acumatica ERP.

Reply:
done